src/qt/transactionfilterproxy.cpp

```diff
@@ -51,8 +52,11 @@ bool TransactionFilterProxy::filterAcceptsRow(int sourceRow, const QModelIndex &
         return false;
     if(datetime < dateFrom || datetime > dateTo)
         return false;
-    if (!address.contains(addrPrefix, Qt::CaseInsensitive) && !label.contains(addrPrefix, Qt::CaseInsensitive))
+    if (!address.contains(m_search_string, Qt::CaseInsensitive) &&
+        !  label.contains(m_search_string, Qt::CaseInsensitive) &&
```

Comment:
clang-format will remove these whitespaces.

Reply:
I consider this a shortcoming of clang-format, then. I don't consider the remaining nits as things that should be changed.